data_pipelines/metadata/cc/task_functions.py

```diff
@@ -17,9 +19,38 @@
 )
 
 
+def get_month_year_french():
+    # Mapping of month numbers to French month names in lowercase
+    month_mapping = {
+        1: "janvier",
+        2: "fevrier",
+        3: "mars",
+        4: "avril",
+        5: "mai",
+        6: "juin",
+        7: "juillet",
+        8: "aout",
+        9: "septembre",
+        10: "octobre",
+        11: "novembre",
+        12: "decembre",
+    }
```

Comment:
Love the magic here  !
Almost certain there is a way to do it with python standard lib but if it works, sounds good as is ?

Reply:
I found a way to do it in english, and in french I'll have to download a oackage `babel` and modify the airflow infra on another repo. Too big of a hassle in my opinion.